tests/test_indieweb_utils.py

```diff
@@ -54,3 +54,22 @@ def test_webpage_feed_discovery(self):
 
         for f in feeds:
             assert f.url in assumed_feeds
+
+
+class TestReplyContextDiscovery:
+    # TODO: Add test cases for Twitter
+    # TODO: Add test cases for other websites
+    # TODO: Add test cases with photo / video responses
+    def test_reply_context_discovery(self):
```

Comment:
I imagine you're planning to use responses to prevent network requests in this test, right?